src/components/MovieCard/MovieCard.jsx

```diff
@@ -1,5 +1,46 @@
 import './MovieCard.scss';
 
-export const MovieCard = () => (
-  <>Put the card here</>
+export const MovieCard = ({ movie: {
+  title,
+  description,
+  imgUrl,
+  imdbUrl,
+  imdbId,
+} }) => (
+  <>
```

Comment:
There is no need to use fragments here because you are returning just one element in your jsx